### ISSUE TITLE
Calculator example gives ENOENT error with MicroPython

### ISSUE THREAD

**Issue by mhsmith** [invent-framework/invent#107]
It shows this in the console, but I don't see any indication of *what* file wasn't found:

```
{
    "name": "ErrnoError",
    "errno": 44,
    "code": "ENOENT",
    "message": "No such file or directory"
}

Ln @ custom.js:167
await in Ln (async)
define @ custom.js:251
(anonymous) @ core.js:291
Promise.then (async)
(anonymous) @ core.js:164
```

If I change `type="mpy"` to `type="py"`, then it works fine.

It looks like this problem was introduced by #104, as the previous main branch commit (7ccfed17458c02ded030545e381da3f90cb8dc6f) works fine.

**Comment by mhsmith** [comment]
Caused by https://github.com/pyscript/pyscript/issues/2035, so should be fixed in PyScript 2024.4.2.

**Comment by mhsmith** [comment]
Unfortunately PyScript 2024.4.2 still has some problems with MicroPython and asyncio:
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "invent/__init__.py", line 21, in <module>
  File "invent/channels.py", line 21, in <module>
  File "invent/task.py", line 5, in <module>
ImportError: no module named 'asyncio'
```

**Comment by mhsmith** [comment]
Fixed in PyScript 2024.5.1.

**Comment by mhsmith** [comment]
Fixed in #114.
